modules/microvm/virtualization/microvm/adminvm.nix

```diff
@@ -55,14 +63,25 @@
         microvm = {
           optimize.enable = true;
           hypervisor = "cloud-hypervisor";
-          shares = [
-            {
-              tag = "ro-store";
-              source = "/nix/store";
-              mountPoint = "/nix/.ro-store";
-              proto = "virtiofs";
-            }
-          ];
+          shares =
+            [
+              {
+                tag = "ro-store";
+                source = "/nix/store";
+                mountPoint = "/nix/.ro-store";
+                proto = "virtiofs";
+              }
+            ]
+            ++ lib.optionals isLoggingEnabled [
+              {
+                # Creating a persistent log-store which is mapped on ghaf-host
```

Comment:
This is only to preserve state across adminvm reboots, correct? Can the comment be extended to mention that?

Reply:
Thanks!